Sequence of Tenses/Yehor_Fedosieiev_sequence_of_tenses.md

```diff
@@ -0,0 +1,14 @@
+6 Сергей сказал Владу, что необходимо будет решить пару организационных вопросов перед пересмотром повышения зарплаты, на что тот ему ответил, что он будет готов обсудить любые варианты.
+>Sergey told Vlad that some organizational questions would have to be resolved before reviewing a salary raise to which he replied that he would be able to discuss any options.
+
+7 Прежде чем приобрести новый компьютер просмотрели огромное количество обзоров и выбрали оптимальный вариант цены качества на рынке.
+>A huge amount of reviews had been watched before purchasing a new computer and the best option on the market was chosen.
+
+8 На пером занятии я их предупредил что им придется много работать самостоятельно, и что им стоит начать читать литературу в оригинале, но они не послушали меня. Человеческую ленность никогда не побороть новым методикам обучения, они могут только её обыграть и вовлечь в процес получения знаний.
+>At the first lesson, I gave them a heads-up that they would have to work hard on their own and they should have to read literature in the original but they disobeyed me. Human laziness can never be overcome by new education methodologies and all they can do is to outplay it and engage into the education process.
+
+9 На прошлом созвоне клиент сказал что подумает над нашим эстимейтом и коммерческим предложением, но он даже не открывал их.
+>At the last call, the customer promised to consider our estimation and the proposal but he has never looked through it.
```

Comment:
At the last call the customer said that he would think over our estimate and commercial offer but he hasn't even opened them.